docs/production-deployment/worker-deployments/worker-versioning.mdx

```diff
@@ -366,14 +368,21 @@ In this scenario, you may also need to use the other [Versioning APIs](/workflow
 
 ## Sunsetting an old Deployment Version
 
-After a version stops accepting new Workflows, it is considered to be "draining," which is displayed in a value called `DrainageStatus`.
-As Workflows pinned to this version complete, the version drains.
+A Worker Deployment Version moves through the following states:
+
+1. **Inactive**: The version exists because a Worker with that version has polled the server. If this version never becomes Active, it will never be Draining or Drained.
+2. **Active**: The version is either Current or Ramping, so it is accepting new workflows and existing auto-upgrade workflows.
+3. **Draining**: The version stopped being Current or Ramping, and it has open pinned workflows running on it. It is possible to be Draining and have no open pinned workflows for a short time, since the drainage status is updated periodically.
+4. **Drained**: The version was draining and now all the pinned workflows that were running on it are closed.
```

Comment:
```suggestion
2. **Active**: The version is either Current or Ramping, so it is accepting new Workflows and existing auto-upgrade Workflows.
3. **Draining**: The version stopped being Current or Ramping, and it has open pinned Workflows running on it. It is possible to be Draining and have no open pinned Workflows for a short time, since the drainage status is updated periodically.
4. **Drained**: The version was draining and now all the pinned Workflows that were running on it are closed.
```